ci-operator/templates/cluster-launch-e2e-openshift-ansible.yaml

```diff
@@ -156,6 +165,48 @@ objects:
         function teardown() {
           set +e
           touch /tmp/shared/exit
+          echo "Gathering artifacts ..."
```

Comment:
This is an exact copy of the other artifact gathering, correct?

Reply:
Yes